### PR TITLE
Transcripts: Style and scroll

### PR DESCRIPTION
**Pull request by SergioEstevao**
Refs #1848 

This PR starts scrolling and style the transcript as the position in the stream moves.

## To test

- Start the app
- Ensure that `transscripts` and `newShowNotesEndpoint ` FF are enabled
- Open an episode with transcripts. Ex: any episode of Cautionary Tales
- Tap play
- Open the player full screen
- Open the transcripts tab
- Check if the transcript display and the text is highlights and scrolls as it's spoken.

## Checklist

- [x] I have considered if this change warrants user-facing release notes and have added them to `CHANGELOG.md` if necessary.
- [x] I have considered adding unit tests for my changes.
- [x] I have updated (or requested that someone edit) [the spreadsheet](https://docs.google.com/spreadsheets/d/107jqrutZhU0fVZJ19SBqxxVKbV2NWSdQC9MFYdLiAxc/edit?usp=sharing) to reflect any new or changed analytics.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR is still a Draft: some checks will be skipped.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by pocketcasts** [comment]
<details><summary>Pocket Casts Prototype Build📲 You can test the changes from this Pull Request in <b>Pocket Casts Prototype Build</b> by scanning the QR code below to install the corresponding build.</summary>
<table>
<tr>
  <td rowspan='6' width='260px'></td>
  <td><b>App Name</b></td><td>Pocket Casts Prototype Build Pocket Casts Prototype Build</td>
</tr>
<tr><td><b>Build Number</b></td><td>pr1867-b5ec9c2</td></tr>
<tr><td><b>Version</b></td><td>7.67</td></tr>
<tr><td><b>Bundle ID</b></td><td>au.com.shiftyjelly.podcasts.prototype</td></tr>
<tr><td><b>Commit</b></td><td>b5ec9c266450b680472a521e32531551a8637e51</td></tr>
<tr><td><b>App Center Build</b></td><td><a href='https://install.appcenter.ms/orgs/automattic/apps/Pocket-Casts-Prototype-Builds/releases/18'>Pocket Casts - Prototype Builds #18</a></td></tr>
</table>
<em>Automatticians: You can use our internal self-serve MC tool to give yourself access to App Center if needed.</em>
</details>

File: podcasts/PlaybackManager.swift [hunk 1]
```diff
@@ -45,7 +45,7 @@ class PlaybackManager: ServerPlaybackDelegate {
     private let shouldDeactivateSession = AtomicBool()
     private var haveCalledPlayerLoad = false
 
-    private let updateTimerInterval = 1 as TimeInterval
+    private let updateTimerInterval = 0.1 as TimeInterval
```
Comment (SergioEstevao) [inline podcasts/PlaybackManager.swift]: Are we not running the risk of flooding the system with to much notifications? Maybe we should consider a different timer and notification for transcript progress updates.

File: Modules/Utils/Sources/PocketCastsUtils/Feature Flags/FeatureFlag.swift [hunk 2]
```diff
@@ -138,7 +138,7 @@ public enum FeatureFlag: String, CaseIterable {
         case .newSharing:
             false
         case .transcripts:
-            false
+            true
```
Comment (leandroalonso) [inline Modules/Utils/Sources/PocketCastsUtils/Feature Flags/FeatureFlag.swift]: We need to revert that before merging. Leaving this message so we won't forget.

File: Modules/Utils/Sources/PocketCastsUtils/Feature Flags/FeatureFlag.swift [hunk 3]
```diff
@@ -98,7 +98,7 @@ public enum FeatureFlag: String, CaseIterable {
         case .endOfYear:
             false
         case .newShowNotesEndpoint:
-            false
+            true
```
Comment (leandroalonso) [inline Modules/Utils/Sources/PocketCastsUtils/Feature Flags/FeatureFlag.swift]: We need to revert that before merging. Leaving this message so we won't forget.

File: podcasts/PlaybackManager.swift (hunk 1 above)
Comment (leandroalonso) [reply podcasts/PlaybackManager.swift]: We need to revert that before merging — this is just for testing purposes. Leaving this message so we won't forget.

**Review by leandroalonso — APPROVED** [review]
We will disable scroll because most transcripts are completely out of sync. However I won't make this change here and this PR is good to go.

**Pull request merged**